docs/source/ko/_toctree.yml

```
@@ -145,8 +145,8 @@
     title: (번역중) bitsandbytes
   - local: in_translation
     title: (번역중) GPTQ
-  - local: in_translation
-    title: (번역중) AWQ
+  - local: tasks/awq.md
```

```suggestion
  - local: quantization/awq.md
```
https://github.com/huggingface/transformers/tree/main/docs/source/en/quantization
원본 문서가 `tasks` 디렉토리가 아니라 `quantization` 폴더 안에 있어서 수정해드려요. 
수정하신 `awq.md` 파일도 위치를 옮겨서 다시 commit 해주셔야 될 것 같아요!
지금까지 리뷰된 부분 commit suggestion(리뷰 반영)한 이후에 위치 옮겨주시면 감사하겠습니다~~!